buildscripts/condarecipe.local/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - {{ compiler('cxx') }}    # [not (armv6l or armv7l or aarch64)]
     # both of these are needed on osx, llvm for the headers, Intel for the lib
     - llvm-openmp              # [osx]
-    - intel-openmp             # [osx]
+    - intel-openmp             # [osx and x86_64]
```

Comment:
xref: https://github.com/numba/numba/issues/7468 there's a reasonably high chance that this dep can just be deleted?